packages/react/src/components/FileUploader/__tests__/FileUploader-test.js

```diff
@@ -50,38 +50,22 @@ describe('FileUploader', () => {
 
   it('should clear all uploaded files when `clearFiles` is called on a ref', () => {
```

Comment:
This unit test seems wrong it was trying to test if remove button works. I reverted back to a version from history that actually test `clearFiles()` function